flocker/docs/bootstrap/_logo.py

```diff
@@ -0,0 +1,51 @@
+
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Logo directive
```

Comment:
This directive should probably just use the helper in simple (it can go there). It was originally written before the helper.